src/bundle/Resources/views/parts/menu/sidebar_base.html.twig

```diff
@@ -34,9 +34,18 @@
 {% endblock %}
 
 {% block label %}
-    {% if item.extras.icon is defined and item.extras.icon != '' %}
-        <svg class="ez-icon ez-icon-{{ item.extras.icon }}">
-            <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#{{ item.extras.icon }}"></use>
+    {% set icon_path = null %}
+    {% set icon_class = (item_class|default('ez-icon ') ~ item.extras.icon_class|default(''))|trim %}
+    {% if item.extras.icon_path is defined and item.extras.icon_path is not empty %}
```

Comment:
```suggestion
    {% if item.extras.icon_path|default %}
```

Reply:
@mikadamczyk Are you sure it will work? Doesn't seem that obvious to me.

Reply:
I tested this in twig and it works for me